opaque_keys/edx/tests/test_locators.py

```diff
@@ -59,3 +61,59 @@ def test_version_tree(self):
         test_id = ObjectId(test_id_loc)
         valid_locator = CourseLocator(version_guid=test_id)
         self.assertEqual(VersionTree(valid_locator).children, [])
+
+
+@ddt.ddt
+class CourseLocatorV2Tests(TestCase):
+    """
+    Tests for :class:`.CourseLocatorV2`
+    """
+
+    def test_course_locator_v2(self):
+        """
+        Verify that the method "from_string" of class "CourseKeyV2"
```

Comment:
Use ` instead of " to format Python entities. This should be fixed in all of your docstrings in this file.